examples/hooks/App.tsx

```diff
@@ -107,7 +107,7 @@ export function App() {
             ]}
           />
 
-          <SearchBox placeholder="Search" />
+          <SearchBox placeholder="Search" autoFocus />
```

Comment:
Should the example have `autoFocus` ? Seems like the other ones don't

Reply:
Since this example is the Hooks playground, I find it useful because you can start typing right away, no need to tab to the input.

No strong opinions on this one.

Reply:
Well it seems handy so I vote to keep it 😄